WcaOnRails/app/views/shared/_wca_id.html.erb

```diff
@@ -2,5 +2,6 @@
 <%# We use a full url because this can be rendered inside of emails -%>
 <% options = {} %>
 <% options[:target] = target if defined?(target) %>
-<%= link_to wca_id, "#{root_url}results/p.php?i=#{wca_id}", options -%>
+<% # FIXME restore single 'root_url' when (if) results is on Rails. %>
+<%= link_to wca_id, "#{root_url(locale:nil)}results/p.php?i=#{wca_id}", options -%>
```

Comment:
Crazy idea: what about actually linking to something like `/fr/results.php?i=2005FLEI01` here, and adding the relevant redirects to our nginx config so that `/fr/results.php?i=2005FLEI01` redirects to `/results.php?i=2005FLEI01`?

(tl;dr: let's do whatever's easier in the long run.)